### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by prajapati-priyanka**
-Add navigation component.
-Can be used for both desktop and mobile.
-Full responsive.

[csb link](https://f4zxtg.csb.app/)

### CODE REVIEW

File: component/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
+.header-container{
+    display: flex;
+    align-items: center;
+    padding: 1.6rem;
+    box-shadow: 0 4px 12px 0 rgb(0,0,0,0.2)
```
Comment (AtulPant2704) [inline component/navigation/navigation.css]: Here, you have used to `rgba(0, 0, 0, 0.2)`

File: component/navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    align-items: center;
+}
+
+@media screen and (max-width: 900px){
+
+    .hamburger-menu{
+        display: block;
+        padding: 0.8rem;
+        padding-right: 1.6rem;
+        /* background-color: red; */
```
Comment (AtulPant2704) [inline component/navigation/navigation.css]: Remove this comment.

**Review by AtulPant2704 — APPROVED** [review]
Rest everything looks good, variable naming is nice.

**Review by mariyasada — APPROVED** [review]
this code looks fine to me.good to merge

**Pull request merged**